plugins/core_user_verify/plugin_class.php

```diff
@@ -20,12 +20,71 @@ public function make_message($message="") {
     }
 
     public function send_email($email, $username, $key, $redirect) {
+        
         $mail = new Mail();
         $mail->addAddress($email,$username);
-        $mail->subject = "User Verification Details for " . Config::sitename();
+        $mail->subject = "Verify Your " . Config::sitename() . " Account";
         $mail->html = "<p>Hello, You have been registered for an account on " . Config::sitename() . "</p>";
-        $mail->html .= "<p>please click here to verify your account: <a href='https://" . ($redirect ?? $_SERVER['SCRIPT_URL']) . "?key=" . $key . "'>Verify</a></p>";
-        $mail->html .= "<p>If you believe this message to be in error or have not signed up for/requested an account, no further action is required</p>";
+
+        // get logos that might be used
+        $custom_logo_id = $this->get_option("logo_image");
+        $admin_logo_id = Configuration::get_configuration_value('general_options','admin_logo');
+
+        // identify which logo to use
+        $site_domain = "https://" . $_SERVER['SERVER_NAME'];
+        if ($custom_logo_id) {
+            $logo_src = $site_domain . "/image/" . $custom_logo_id;
+        }
+        else if ($admin_logo_id) {
+            $logo_src = $site_domain . "/image/" . $admin_logo_id;
+        }
+        else {
+            $logo_src = $site_domain . "/admin/templates/clean/alba_logo.webp";
+        }
+        
+        // set link and cta colors
+        $config_link_color = $this->get_option("email_link_color");
```

Comment:
` $config_link_color = $this->get_option("email_link_color") ?? "color";` to prevent null errors